featuretools/entityset/entity.py

```diff
@@ -591,25 +591,19 @@ def _filter_and_sort(self, df, time_last=None,
         """
         if self.time_index:
             if time_last is not None and not df.empty:
```

Comment:
is it possible for time last to be none by this point in the code? If not, let's remove

Reply:
`Entity.get_all_instances` and a few others call this without specifying a time last